packages/forms/src/directives/shared.ts

```diff
@@ -36,7 +52,10 @@ export function controlPath(name: string|null, parent: ControlContainer): string
  * @param control Form control instance that should be linked.
  * @param dir Directive that should be linked with a given control.
  */
-export function setUpControl(control: FormControl, dir: NgControl): void {
+export function setUpControl(
+    control: FormControl, dir: NgControl,
+    callSetDisabledState: SetDisabledStateOption|null = setDisabledStateDefault): void {
+  if (callSetDisabledState == null) callSetDisabledState = setDisabledStateDefault;
```

Comment:
```suggestion
  if (callSetDisabledState === null) callSetDisabledState = setDisabledStateDefault;
```
(unless `==` was intentional to cover the `undefined` case as well)